.github/workflows/security.yml

```diff
@@ -72,26 +72,29 @@ jobs:
     needs: trivy-repo
     strategy:
       matrix:
-        version:
-          - "latest"
+        include:
+          - channel: "3/edge"
```

Comment:
`3/edge` doesn't yet exist. Should we comment it out for now?

Reply:
It seems 3/edge does exist:

```
$ snap info microcloud | grep 3/edge
3/edge:           git-ba9113e   2024-11-12 (1125) 10MB -
```

Reply:
You are right, my mistake, I meant `3/stable`.